server/fleet/hosts.go

```diff
@@ -75,6 +75,8 @@ func (s MacOSSettingsStatus) IsValid() bool {
 // - GET /hosts/count (count hosts, which calls svc.CountHosts or svc.CountHostsInLabel)
 // - GET /labels/{id}/hosts (list hosts in label)
 // - GET /hosts/report
+// - POST /hosts/delete (calls svc.hostIDsFromFilters)
+// - POST /hosts/transfer/filter (calls svc.hostIDsFromFilters)
```

Comment:
That reminds me, the API docs should probably be updated to mention that the macos settings filter does not apply to "all teams", always to "no team"?

Reply:
Yep! This was included in the prior PR.